android/app/src/main/java/net/skyscanner/backpack/MainApplication.kt

```diff
@@ -49,6 +50,7 @@ class MainApplication : Application(), ReactApplication {
     override fun onCreate() {
         super.onCreate()
         AndroidThreeTen.init(this)
+        BackpackReactNative.init(this)
```

Comment:
This is what we will use in the app as well, it has the added benefit of us using the same mechanism and potentially finding problems before they happen in the app.